main/src/ca/uwaterloo/flix/api/lsp/Entity.scala

```diff
@@ -20,6 +20,11 @@ import ca.uwaterloo.flix.language.ast.{Name, SourceLocation, Symbol, TypedAst}
 
 sealed trait Entity {
   def loc: SourceLocation
+
+  def isInRange(range: Range): Boolean = {
```

Comment:
Could you I ask you to add a comment? Does it mean that `range` is contained with entity or part of range is or what? :)

Reply:
This calculates wether the entity is fully included in a range, which can be a multiline range.

Reply:
OK, can you you try to add comment? :)

Reply:
should be there already :)

Reply:
I meant something like:

```
/**
  * Returns `true` if the given range `range` is fully included in `this` entity (i.e. the given range must starter later and end earlier.)
  */
```

Reply:
ok!